File: DayOne/Python/LevelOne/perfect_square_array_test.py
from unittest import TestCase

from perfect_square_array import *

class TestPerfectSquareArray(TestCase):

    def test_that_i_have_an_array_and_returns_elements_that_are_perfect_square(self):

        array =  [4, 7, 9, 10, 16, 18]

        array_check = [4, 9, 16]

        self.assertEqual(array_check,perfect_square_check(array))

        

File: DayOne/Java/LevelOne/PerfectSquareArrayTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class PerfectSquareArrayTest{

    @Test
    public void testThatIHaveAnArrayAndIgetElementsofArrayThatArePerfectSquares(){

    int [] array = {4, 7, 9, 10, 16, 18};

    double [] arrayCheck = {4.0,9.0,16.0};

    double [] actual = PerfectSquareArray.perfectSquareCheck(array);

    assertArrayEquals(actual,arrayCheck);
    }

}
